core/com.b2international.snowowl.core.rest.tests/src/com/b2international/snowowl/core/rest/bundle/BundleRestApiTest.java

```diff
@@ -0,0 +1,120 @@
+/*
+ * Copyright 2021 B2i Healthcare Pte Ltd, http://b2i.sg
+ * 
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.b2international.snowowl.core.rest.bundle;
+
+import static com.b2international.snowowl.core.rest.BunldeApiAssert.assertBundleCreated;
+import static com.b2international.snowowl.core.rest.BunldeApiAssert.assertBundleGet;
+import static com.b2international.snowowl.core.rest.BunldeApiAssert.assertBundleSearch;
+import static com.b2international.snowowl.core.rest.BunldeApiAssert.assertCreate;
+import static com.b2international.snowowl.core.rest.BunldeApiAssert.prepareCreateRequestBody;
+import static org.hamcrest.CoreMatchers.equalTo;
+import static org.hamcrest.CoreMatchers.hasItem;
+import static org.hamcrest.collection.IsEmptyCollection.empty;
+import static org.hamcrest.collection.IsMapContaining.hasEntry;
+
+import java.util.List;
+import java.util.Map;
+
+import org.junit.Test;
+
+import com.b2international.commons.json.Json;
+import com.b2international.snowowl.core.internal.ResourceDocument;
+import com.fasterxml.jackson.core.JsonProcessingException;
+
+/**
+ * @since 8.0
+ */
+public class BundleRestApiTest {
+	
+	@Test
+	public void createBundleNoId() {
+		assertCreate(
+				Json.object())
+		.statusCode(400).body("violations", hasItem("'id' may not be empty (was 'null')"));
+	}
+
+	@Test
+	public void createBundleNoTitle() {
+		assertCreate(
+				Json.object(
+					ResourceDocument.Fields.ID, "b1"))
+		.statusCode(400).body("violations", hasItem("'title' may not be empty (was 'null')"));
+	}
+
+	@Test
+	public void createBundleNoUrl() {
+		assertCreate(
+				Json.object(
+						ResourceDocument.Fields.ID, "b1",
+						ResourceDocument.Fields.TITLE, "Bundle title b2"))
+		.statusCode(400).body("violations", hasItem("'url' may not be empty (was 'null')"));
+	}
+	
+	@Test
+	public void createBundle() throws JsonProcessingException {
+		final Json body = prepareCreateRequestBody("b1");
+		assertBundleCreated(body);
+		assertBundleGet("b1").and()
```

Comment:
The first assert should be a `statusCode` assert in any REST API test. Please add it to here and all other methods.